scripts/copy-cred-helper.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+OS="$1"
+ARCH="$2"
+
+case "$OS" in
+	"darwin")
+		case "$ARCH" in
+			"amd64")
+				cp binaries/gptscript-credential-osxkeychain releases/default_darwin_amd64_v1
+				;;
+			"arm64")
+				cp binaries/gptscript-credential-osxkeychain releases/default_darwin_arm64
+				;;
+		esac
+		;;
+	"windows")
+		case "$ARCH" in
+			"amd64")
```

Comment:
We did add `arm64` binary for Windows. Should we handle that here?

Reply:
Oh, yes I will be sure to add that. Thanks

Reply:
Added